### ISSUE TITLE
 ef core

### ISSUE THREAD

**Issue by zhujiancc** [JasperFx/marten#1138]
Can I integrated marten to  ef core , so that I can both use ef and marten's jsonb for pgsql

**Comment by oskardudycz** [comment]
Yes, you can. For more detailed answer, I'd need more detailed question :) I was using both of them and I didn't find any bigger issues or blockers
